### PR TITLE
Tighter grid on article layouts

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Prevent horizontal overflow on `GridItem`.

## Why?

An explicit width on a child of the `.ad-slot-container` can break our layout if it is wider than 100%.

## Screenshots

| Before      | After      |
| ----------- | ---------- |
| ![before][] | ![after][] |
| image | image | 

[before]: https://github.com/guardian/dotcom-rendering/assets/76776/85dcf05f-5970-4169-96e4-1c19374bddf7
[after]: https://github.com/guardian/dotcom-rendering/assets/76776/9b744f87-c3dd-4063-8bb3-3f15bbc2594e

### CODE REVIEW

**Review by JamieB-gu — APPROVED** [review]

File: dotcom-rendering/src/lib/rootStyles.ts [hunk 1]
```diff
@@ -46,4 +46,9 @@ export const rootStyles = (
 		background: ${sourcePalette.brandAlt[400]};
 		color: ${sourcePalette.neutral[7]};
 	}
+
+	.ad-slot-container {
+		/* prevent third-party code from breaking our layout */
+		overflow-x: hidden;
+	}
```
Comment (mxdvl) [inline dotcom-rendering/src/lib/rootStyles.ts:53]: @guardian/commercial-dev could you confirm that this won’t prevent fluid ads being as wide as necessary?

**Review by JamieB-gu — APPROVED** [review]

File: dotcom-rendering/src/layouts/CommentLayout.tsx [hunk 2]
```diff
@@ -90,26 +90,26 @@ const StandardGrid = ({
 					Right Column
 				*/
 				${from.wide} {
-					grid-template-columns: 219px 1px 1fr 300px;
+					grid-template-columns: 219px 1px 620px 60px 300px;
```
Comment (mxdvl) [inline dotcom-rendering/src/layouts/CommentLayout.tsx:93]:
```ts
const getWidth = (
	columns: 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14,
) => 60 * columns + 20 * (columns - 1);
```

Comment (JamieB-gu) [reply dotcom-rendering/src/layouts/CommentLayout.tsx:93]: https://github.com/guardian/dotcom-rendering/blob/5aeec757efa9d918802e999bd503fe3cd22eb065/dotcom-rendering/.storybook/decorators/grid.ts#L29-L30

Comment (mxdvl) [reply dotcom-rendering/src/layouts/CommentLayout.tsx:93]: Exactly @JamieB-gu! 🚀

**Review by JamieB-gu — APPROVED** [review]

**Pull request merged**